build/configs/artik053/tc/defconfig

```diff
@@ -242,7 +242,7 @@ CONFIG_ARTIK05X_FLASH_PART=y
 CONFIG_ARTIK05X_FLASH_MINOR=0
 CONFIG_ARTIK05X_FLASH_PART_LIST="16,48,192,32,512,2400,1536,1536,1400,8,512,"
 CONFIG_ARTIK05X_FLASH_PART_TYPE="none,ftl,none,none,none,none,none,ftl,smartfs,config,none,"
-CONFIG_ARTIK05X_FLASH_PART_NAME="bl1,sssro,bl2,sssfw,wlanfw,os,factory,ota,user,nvram,sssrw,"
+CONFIG_ARTIK05X_FLASH_PART_NAME="bl1,rom,bl2,sssfw,wlanfw,os,factory,ota,user,nvram,sssrw,"
```

Comment:
Why do you modify this in this PR?

Reply:
To enable fs_helper config, you need the partition 'rom' area.
So I changed the sssro area which I thought would not be used in tc config to rom.

Reply:
Then, why don't you split user area instead of sssro? defconfig is not being to support your usages. How do you confirm tc does not need sssro? Our guidance also say splitting user area. If you have more clear reason to use ssro area, please let me know.

Reply:
modify 1400                 1000,400
            smartfs   ==>   smarfs,romfs
            user                  user,rom